prinz_sample/src/main/scala/pl/touk/nussknacker/prinz/sample/SampleConfigCreator.scala

```diff
@@ -11,15 +13,19 @@ class SampleConfigCreator extends EmptyProcessConfigCreator {
 
   protected def allCategories[T](obj: T): WithCategories[T] = WithCategories(obj, "FraudDetection", "Recommendations")
 
+  override def services(processObjectDependencies: ProcessObjectDependencies): Map[String, WithCategories[Service]] = Map(
+    "prinzEnricher" -> allCategories(new PrinzEnricher)
+  )
+
   override def sourceFactories(processObjectDependencies: ProcessObjectDependencies): Map[String, WithCategories[SourceFactory[_]]] = Map(
     "periodic" -> allCategories(PeriodicSourceFactory)
   )
 
   override def sinkFactories(processObjectDependencies: ProcessObjectDependencies): Map[String, WithCategories[SinkFactory]] = Map(
-    "empty" -> allCategories(SinkFactory.noParam(EmptySink))
+    "empty" -> allCategories(SinkFactory.noParam(EmptySink)),
+    "logging" -> allCategories(SinkFactory.noParam(LoggingSink))
```

Comment:
why do we need logging sink in nussknacker?

Reply:
removed